src/mgos_barometer_bme280.c

```diff
@@ -74,19 +79,51 @@ bool mgos_barometer_bme280_create(struct mgos_barometer *dev) {
     return false;
   }
 
+  if( dev->capabilities && MGOS_BAROMETER_CAP_HYGROMETER ) {
+    uint8_t reg_data[ 7 ];
+    int16_t dig_h4_lsb;
+    int16_t dig_h4_msb;
+    int16_t dig_h5_lsb;
+    int16_t dig_h5_msb;
+
+    if (!mgos_i2c_read_reg_n(dev->i2c, dev->i2caddr, BME280_REG_HUMIDITY_CALIB_ADDR, 7, reg_data)) {
+      free(dev->user_data);
+      return false;
+    }
+
+    bme280_data->calib.dig_H2 = (int16_t)BME280_CONCAT_BYTES(reg_data[1], reg_data[0]);
+    bme280_data->calib.dig_H3 = reg_data[2];
+    dig_h4_msb = (int16_t)(int8_t)reg_data[3] * 16;
+    dig_h4_lsb = (int16_t)(reg_data[4] & 0x0F);
+    bme280_data->calib.dig_H4 = dig_h4_msb | dig_h4_lsb;
+    dig_h5_msb = (int16_t)(int8_t)reg_data[5] * 16;
+    dig_h5_lsb = (int16_t)(reg_data[4] >> 4);
+    bme280_data->calib.dig_H5 = dig_h5_msb | dig_h5_lsb;
+    bme280_data->calib.dig_H6 = (int8_t)reg_data[6];
+  }
+
   // SPI | 0.5ms period | 16X IIR filter
-  if (!mgos_i2c_write_reg_b(dev->i2c, dev->i2caddr, BME280_REG_CONFIG, 0x00 | BME280_STANDBY_500us << 2 | BME280_FILTER_16X << 5)) {
+  if (!mgos_i2c_write_reg_b(dev->i2c, dev->i2caddr, BME280_REG_CONFIG, 0x00 | BME280_STANDBY_500us << 2 | BME280_FILTER_2X << 5)) {
```

Comment:
why change to 2x filter here?

Reply:
Don't recollect why so I would not insist on it.